debian/appscale_build.sh

```diff
@@ -1,7 +1,31 @@
 #!/bin/bash
+#
+# This script install appscale on the local machine. It pulls in all the
```

Comment:
install -> installs. Also, probably AppScale for consistency with the next line.